src/pybind/mgr/cephadm/module.py

```diff
@@ -131,14 +131,18 @@ def __init__(self, mgr):
         # type: (CephadmOrchestrator) -> None
         self.mgr = mgr
         self.specs = {} # type: Dict[str, orchestrator.ServiceSpec]
+        self.spec_created = {} # type: Dict[str, datetime.datetime]
 
     def load(self):
         # type: () -> None
         for k, v in six.iteritems(self.mgr.get_store_prefix(SPEC_STORE_PREFIX)):
             service_name = k[len(SPEC_STORE_PREFIX):]
             try:
-                spec = ServiceSpec.from_json(json.loads(v))
+                v = json.loads(v)
+                spec = ServiceSpec.from_json(v['spec'])
```

Comment:
We actually want the spec _and_ the placement. -> `spec = ServiceSpec.from_json(v)`

Reply:
oh, nvm.. I should've read the new `save()` method